### ISSUE TITLE
Access Violation Error on BW halfway through downloading a manga. 

### ISSUE THREAD

**Issue by NightoVisi0n** [xuzhengyi1995/Manga_downloader#109]
I have two books in my BW bookshelf, the first one downloaded without a hitch when using the new method, but when I tried to download a newly bought manga, around page 21 I suddenly got cut off by BW viewer saying the error code "STATUS_ACCESS_VIOLATION" like this:

![IMG_20220921_101016_916](https://user-images.githubusercontent.com/114042569/191406459-a41be322-65c9-4e79-a52f-07c17cf9e146.jpg)

I had reset the reading status back to 0, cleared all cache and data, re-login, and even rebooted my pc, it's still persistent. Have BW finally caught up on what's happening, something wrong from my part, or something else?

**Comment by xuzhengyi1995** [comment]
Could you go to `C:\bw-downloader-profile\Crashpad\reports` and see is there any `*.dmp` files, if there is, attach it could be really helpful. Give the manga url could also help. This is not BW's problem, it is a chromium renderer process crash and should be investigated. Thanks!

**Comment by NightoVisi0n** [comment]
Yeah, there is. https://drive.google.com/file/d/15M9tne8oQXYOgvLpwO9eRnLONnC_2yd3/view?usp=sharing (i can't attach the file directly here)

**Comment by xuzhengyi1995** [comment]
Thanks for the dump file, it is a memory null pointer problem, if this could reproduce, could you give the manga URL so I can investigate this crash ?

Is this the only one or you have seen any other URL that will crash the renderer? Thanks.

**Comment by NightoVisi0n** [comment]
https://bookwalker.jp/ded383ebf1-291f-40d6-bef3-fe6c510c3d43/

I was able to reproduce the error one last time (as in tried to download the same manga again but got cut off on page 5 instead of 21), after that, I accidentaly updated Windows , and now I'm completely unable to download any. I did a fresh new install. The console script runs perfectly fine, but it refuses to automatically create the **bw_export_data** folder and in turn the random folder needed to save the images. Tried two different accounts but to no luck.

**Comment by xuzhengyi1995** [comment]
> https://member.bookwalker.jp/app/03/webstore/cooperation?r=BROWSER_VIEWER/d383ebf1-291f-40d6-bef3-fe6c510c3d43/https%3A%2F%2Fbookwalker.jp%2FholdBooks%2F / https://bookwalker.jp/ded383ebf1-291f-40d6-bef3-fe6c510c3d43/
> 
> I was able to reproduce the error one last time (as in tried to download the same manga again but got cut off on page 5 instead of 21), after that, I accidentaly updated Windows , and now I'm completely unable to download any. I did a fresh new install. The console script runs perfectly fine, but it refuses to automatically create the **bw_export_data** folder and in turn the random folder needed to save the images. Tried two different accounts but to no luck.

That is strange, one possible problem is have you start it with `--no-sandbox` ?

**Comment by NightoVisi0n** [comment]
I always copy-paste the commands and triple-check before entering, so I think it's not a mistyped command sort of problem from my part. Someone posted another issue recently, after updating their win 11 to 22h2, their bw_export_data also stopped working as intended like me. Maybe it's from there?
